comb_spec_searcher/equiv_db.py

```diff
@@ -70,16 +71,60 @@ def __getitem__(self, comb_class: int) -> int:
             self.parents[ancestor] = root
         return root
 
-    def add_edge(self, label: int, other_label: int) -> None:
+    def get_one_way_vertices(self) -> Dict[int, Set[int]]:
```

Comment:
Maybe a doctstring. I don't find the name super descriptive

Reply:
Fair, this is important for the tidy up.